standards-in-genomic-sciences.csl

```diff
@@ -182,7 +182,7 @@
           <text macro="editor" prefix=" " suffix="."/>
           <group prefix=" " suffix=".">
             <text variable="container-title" font-style="italic" form="short" suffix=" "/>
-            <group delimiter=";" prefix=" ">
+            <group delimiter=";">
               <date variable="issued">
                 <date-part name="year"/>
               </date>
```

Comment:
1) Finding the relevant cases might be a different. I would guess that for that you need to go trough the XML structure and looking for group nodes with only one child. (Sure, this is doable, but it is different from find&replace of a regexp).
2) You are right. That was a false thought from my side...
3) One would have to merge all attributes from the group and its child node. I am not sure if this is possible automatically.